### DISCUSSION TITLE
demo提供的11.jpg图片，识别不全。想修改dt_boxes num 和  rec_res num参数大小

### DISCUSSION THREAD

**Discussion by freezehe** [PaddlePaddle/PaddleOCR#13258] [Q&A]
如题：我服务器上部署的是v4模型版本，日志打印默认这两个参数大小是16，导致demo提供的11.jpg图片都没有识别全文字。
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/11324122/c6a1fbd9-c723-4b36-9f3b-1ad07d591385)
然后我在windows上参考了你们提供的python代码，这几个参数大小是24。
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/11324122/c00a8ee6-3949-4fcb-bd14-66e50aea28fc)
如何设置这些参数？是否有技术群可以讨论交流

**Comment by UserWangZz** [comment]
这两个是模型检测到的检测框和识别到的文本的个数，不是参数
